src/common/components/cards/card-kebab-menu-button.tsx

```diff
@@ -170,11 +162,15 @@ export class CardKebabMenuButton extends React.Component<CardKebabMenuButtonProp
         this.setState({ showNeedsSettingsContent: true });
     }
 
-    private openDropdown = (event): void => {
-        this.setState({ target: event.currentTarget, isContextMenuVisible: true });
-    };
+    private showToastWithSuccessMessage(): void {
+        this.setState({ showingCopyToast: true, toastText: 'Failure details copied.' });
+    }
+
+    private hideToast(): void {
+        this.setState({ showingCopyToast: false, toastText: '' });
```

Comment:
Does the toast's disappearance involve a fade out animation, or is it immediate? If it isn't immediate, we might not want to empty the text here

Reply:
I tried remove the text state change here, there is no change to the dismiss animation. So my understanding is that there is no fade out animation. 
Also, compared with master, not seeing fade out animation.